### PR TITLE
Add support for Cypress 13

### LINKED ISSUE #142: conflicting dependencies cypress-drag-drop v2.2.4 with cypress v13.0.0
Hi,

We would like to upgrade cypress v13.0.0, but experiencing some conflicting dependencies together with the latest version of cypress-drag-drop (2.2.4).

Would it be possible to release an updated version of cypress-drag-drop, which is compatible with cypress 13.0.0?

Thanks in advance! 
Cheers

### PR DESCRIPTION
**Pull request by jonkoops**
Adds Cypress 13 as a supported version, closes #142.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,6 @@
     "webpack-dev-server": "^3.11.0"
   },
   "peerDependencies": {
-    "cypress": "^2.1.0 || ^3.1.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
+    "cypress": "^2.1 || ^3.1 || 4 - 13"
```
Comment (bierik) [inline package.json:51]: I like the version range annotation. Is it also possible to do `2 - 13`? Or is it necessary that `2.1` and `3.1` are listed separately?

**Review by bierik — CHANGES_REQUESTED** [review]
Apparently Cypress 13 is no longer working with node 14. The matrix build under https://github.com/4teamwork/cypress-drag-drop/blob/master/.github/workflows/test.yml#L11 is still referencing node 14, that is why the tests are failing. Can you please remove node version 14 and probably add version 18 instead in the matrix build. Thank you so much for your contribution.

**Comment by jonkoops** [comment]
> Can you please remove node version 14 and probably add version 18 instead in the matrix build.

Done :heavy_check_mark:

File: package.json (hunk 1 above)
Comment (bierik) [reply package.json:51]: I think the explicit support for `2.1.x` and `3.1.x` was introduced by accident a long time ago. IMO having `2 - 13` is much better. Thank you for pointing that out 😃.

Comment (jonkoops) [reply package.json:51]: Upon closer consideration, perhaps it is time to start dropping support for these older versions? They are not actually being tested against on CI, which makes it highly doubtful that this library is actually still compatible with these older versions.

Then again I could be wrong about that, I have no idea how much (or little) the API for Cypress has changed. Perhaps this is also better served as a separate issue/discussion.

**Comment by jonkoops** [comment]
Hmmm, looks like the version of WebPack that is being used there is not compatible with Node.js version 18. Perhaps it would be best to handle this in a separate PR, I'll remove version 18 from the matrix for now.

File: package.json (hunk 1 above)
Comment (jonkoops) [reply package.json]: Great, updated the version range as discussed :+1:

**Comment by hooman-mirghasemi** [comment]
Hi, When new version of this package will release with cypress 13 support?

**Comment by TheZoker** [comment]
A release would be really appriciated!

**Comment by lvonlanthen** [comment]
Version `2.2.5` with support for Cypress 13.x was released just now.

**Comment by rousah** [comment]
thanks!

**Pull request merged**